From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Stultz <john.stultz@linaro.org>
Date: Tue, 23 Feb 2021 00:01:34 +0000
Subject: ANDROID: db845c_gki.fragment: Remove QCOM_GENI_CONSOLE modules now
 they are built in

With commit a7961b063571e ("ANDROID: gki_defconfig: Enable QCOM geni console")
the QCOM_GENI options are now built in. Trying to load the modules built in the
db845c_gki.fragment cause boot failure, so remove them from the config fragment.

Fixes: a7961b063571e ("ANDROID: gki_defconfig: Enable QCOM geni console")
Signed-off-by: John Stultz <john.stultz@linaro.org>
Change-Id: I9840d66366bfc489dc8e546277f38936c6c16186
---
 arch/arm64/configs/db845c_gki.fragment | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/configs/db845c_gki.fragment b/arch/arm64/configs/db845c_gki.fragment
--- a/arch/arm64/configs/db845c_gki.fragment
+++ b/arch/arm64/configs/db845c_gki.fragment
@@ -5,8 +5,6 @@ CONFIG_USB_NET_AX8817X=m
 CONFIG_USB_NET_AX88179_178A=m
 CONFIG_INPUT_PM8941_PWRKEY=m
 CONFIG_SERIAL_MSM=m
-CONFIG_SERIAL_QCOM_GENI=m
-CONFIG_SERIAL_QCOM_GENI_CONSOLE=y
 CONFIG_I2C_QCOM_GENI=m
 CONFIG_I2C_QUP=m
 CONFIG_PINCTRL_MSM=m
@@ -40,7 +38,6 @@ CONFIG_SDM_GPUCC_845=m
 CONFIG_QCOM_CLK_RPMH=m
 CONFIG_SDM_DISPCC_845=m
 CONFIG_HWSPINLOCK_QCOM=m
-CONFIG_QCOM_GENI_SE=m
 CONFIG_QCOM_LLCC=m
 CONFIG_QCOM_RMTFS_MEM=m
 CONFIG_QCOM_SMEM=m
